source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardTemplate.cs

```diff
@@ -4,6 +4,7 @@
 using Antlr4.Runtime.Tree;
 using Newtonsoft.Json;
 using System;
+using System.Collections;
 
```

Comment:
Don't forget to update document for this breaking change

Reply:
I'll be sure to note this in the release notes. Thanks!